dvc/dagascii.py

```diff
@@ -61,99 +63,14 @@ def __init__(self, cols, lines):
     def draw(self):
         """Draws ASCII canvas on the screen."""
         if sys.stdout.isatty():  # pragma: no cover
-            from asciimatics.screen import Screen
-
-            Screen.wrapper(self._do_draw)
+            lines = []
+            for line in self.canvas:
+                lines.append("".join(line))
+            pydoc.pager(os.linesep.join(lines))
         else:
             for line in self.canvas:
                 print("".join(line))
 
-    def _do_draw(self, screen):  # pragma: no cover
```

Comment:
this is not used now, however does this class make sense now without it?
Anyway i would remove it.